aws-android-sdk-kinesis/src/test/java/com/amazonaws/mobileconnectors/kinesis/kinesisrecorder/KinesisRecorderTest.java

```diff
@@ -40,7 +40,7 @@
 import java.util.UUID;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(manifest = Config.NONE)
+@Config(manifest = Config.NONE, sdk = 27)
```

Comment:
You can make a single `src/test/resources/robolectric.properties` that includes these values and then you don’t have to put them on every test.

Reply:
agreed! Will do this for mobileanalytics also